.github/workflows/clone_with_pinned_version.yml

```diff
@@ -0,0 +1,49 @@
+name: JEDI Bundle Pinned Version Clone Test
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+
+jobs:
+  buildbundle:
+    name: Clone oops bundle
+    runs-on: ubuntu-latest
+
+    steps:
+
+    # Setup Python
+    - name: Set up Python 3.9
```

Comment:
Pinning to Python 3.9 is OK, but I would suggest we test against more Python versions to future proof ourselves against breaking changes. Note that it's possible to configure builds to only require certain builds to pass for success.

The best way to test against multiple Python versions is to use a matrix build, as described here:

https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python#using-multiple-python-versions